modules/nixos/common/default.nix

```diff
@@ -12,6 +12,9 @@
     inputs.srvos.nixosModules.server
   ];
 
+  # breaks smartctl in the srvos telegraf mixin
+  security.sudo.execWheelOnly = lib.mkForce false;
```

Comment:
@zimbatm makes me wonder if we should have this in srvos than.
It seems to be a common pattern to grant applications access to certain white-listed commands with sudo. Alternative would be to make it a setuid wrapper that only telegraf can execute.